android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -557,6 +556,12 @@ protected TiUIView handleGetView()
 			}
 
 			Activity activity = getActivity();
+			if (activity.isDestroyed()) {
+				activity = activity.getParent();
```

Comment:
Hold on. I see a problem here. The parent activity can be in a destroyed state too.

Instead of falling-back to the parent activity, I think you should set the activity to null here and let the below code grab the top-most activity. Odds are the top-most activity is the one the proxy's view is going to be used on. Or alternatively, walk up the parent activity tree until you find a non-destroyed activity, such as the LauncherActivity.

The below code should exercise this issue...
```javascript
var rootWindow = Ti.UI.createWindow();
var button = Ti.UI.createButton({ title: "Start Test" });
button.addEventListener("click", function(e) {
	var imageView;
	var childWindow1 = Ti.UI.createWindow();
	childWindow1.addEventListener("open", function(e) {
		var childWindow2 = Ti.UI.createWindow();
		childWindow2.addEventListener("open", function(e) {
			// Create the image view under the 2nd child window.
			imageView = Ti.UI.createImageView();
			childWindow2.close();
		});
		childWindow2.addEventListener("close", function(e) {
			childWindow1.close();
		});
		childWindow2.open();
	});
	childWindow1.addEventListener("close", function(e) {
		// Attempt to access after both child windows have been destroyed.
		imageView.tintColor;
	});
	childWindow1.open();
});
rootWindow.add(button);
rootWindow.open();
```